pkg/controller/master/backup/backup_target.go

```diff
@@ -38,9 +29,18 @@ const (
 
 	longhornBackupTargetSettingName       = "backup-target"
 	longhornBackupTargetSecretSettingName = "backup-target-credential-secret"
+
+	AWSAccessKey       = "AWS_ACCESS_KEY_ID"
+	AWSSecretKey       = "AWS_SECRET_ACCESS_KEY"
+	AWSEndpoints       = "AWS_ENDPOINTS"
+	AWSCERT            = "AWS_CERT"
+	VirtualHostedStyle = "VIRTUAL_HOSTED_STYLE"
+	HttpProxy          = "HTTP_PROXY"
+	HttpsProxy         = "HTTPS_PROXY"
+	NoProxy            = "NO_PROXY"
```

Comment:
Where do we use these constants

Reply:
Sorry, I forget to delete them. Thank you.